src/Npgsql/TypeHandlers/HstoreHandler.cs

```diff
@@ -181,6 +183,29 @@ async ValueTask<T> ReadInto<T>(T dictionary, int numElements, NpgsqlReadBuffer b
         }
 
         #endregion
+
+        #region IReadOnlyDictionary
+
+        /// <inheritdoc />
+        public int ValidateAndGetLength(
+            IReadOnlyDictionary<string, string?> value, ref NpgsqlLengthCache? lengthCache, NpgsqlParameter? parameter)
```

Comment:
Move parameter to the same line as the method name here and below.

If you implement interface explicitly, there will be now casts above. This is preferable way since `IReadOnlyDictionary<,>` is not a default type to handle.